web-wallet/src/lib/dusk/components/CopyField/CopyField.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+  import { mdiContentCopy } from "@mdi/js";
+  import { Button, Textbox } from "$lib/dusk/components";
+  import { toast } from "$lib/dusk/components/Toast/store";
+
+  /** @type {string} */
+  export let name;
+
+  /** @type {string} */
+  export let displayValue;
+
+  /** @type {string} */
+  export let rawValue;
+
+  /** @type {boolean} */
+  export let disabled = false;
+
+  function copyToClipboard() {
+    navigator.clipboard.writeText(rawValue);
+    toast("success", `${name} copied`, mdiContentCopy);
+  }
+</script>
+
+<div class="copy-field">
+  <Textbox
+    className="copy-field__content"
+    value={displayValue}
+    type="text"
+    readOnly
+  />
+  <Button
+    aria-label="Copy Address"
+    className="copy-field__button"
+    icon={{ path: mdiContentCopy }}
+    on:click={copyToClipboard}
+    variant="primary"
+    {disabled}
+  />
+</div>
+
+<style lang="postcss">
+  .copy-field {
+    display: flex;
+    align-items: center;
+    gap: var(--default-gap);
+  }
+  :global {
```

Comment:
why not make the whole set of rules global?

Reply:
I have moved it to a CopyField.css file, similarly to how we have it for other components.